spec/models/news/importer_spec.rb

```diff
@@ -0,0 +1,116 @@
+require 'rails_helper'
+
+RSpec.describe News::Importer do
+  subject(:instance) { described_class.new json_data }
+
+  let(:json_data) { file_fixture 'news/govuk_stories.json' }
+
+  describe '.new' do
+    it { is_expected.to be_instance_of described_class }
+  end
+
+  describe '#stories' do
+    subject { instance.stories }
+
+    it { is_expected.to have_attributes length: 3 }
+  end
+
+  describe '#import!' do
+    it { expect { instance.import! }.to change(News::Item, :count).by(3) }
+    it { expect(instance.import!).to be 3 }
+
+    context 'with collection names' do
+      subject { News::Collection.all.pluck(:name) }
+
+      before { instance.import! }
+
+      it { is_expected.to include 'Tariff notices' }
+      it { is_expected.to include 'Tariff stop press' }
+      it { is_expected.to include 'Trade news' }
+      it { is_expected.to include 'Service updates' }
+    end
+
+    context 'with first story' do
+      subject(:first_item) { News::Item.order(:id).first }
+
+      before do
+        freeze_time
+        instance.import!
+      end
+
+      it { is_expected.to have_attributes title: /four-wheeled/ }
+      it { is_expected.to have_attributes slug: /a-four-wheeled/ }
+      it { is_expected.to have_attributes precis: /classification for a four-wheeled/ }
+      it { is_expected.to have_attributes content: /^## New regulation/ }
+      it { is_expected.to have_attributes start_date: Date.parse('2022-03-27') }
+      it { is_expected.to have_attributes end_date: nil }
+      it { is_expected.to have_attributes show_on_uk: true }
+      it { is_expected.to have_attributes show_on_xi: true }
+      it { is_expected.to have_attributes show_on_updates_page: true }
+      it { is_expected.to have_attributes show_on_banner: false }
+      it { is_expected.to have_attributes show_on_home_page: false }
+      it { is_expected.to have_attributes imported_at: Time.zone.now }
+
+      context 'with collections' do
+        subject { first_item.collections }
+
+        it { is_expected.to have_attributes length: 1 }
+        it { is_expected.to all have_attributes name: 'Tariff notices' }
+      end
+    end
+
+    context 'with collection which does not exists' do
+      subject { News::Collection.all.pluck(:name) }
+
+      before { instance.import! }
+
+      let(:json_data) { StringIO.new incomplete_data.to_json }
+
+      let :incomplete_data do
```

Comment:
Which piece of data is missing?
if only one field is missing,  we could  specify which one is missing.  

For example:
let(:news_with_missing_slug)